api/jobs.yaml

```diff
@@ -200,6 +200,9 @@ definitions:
         type: array
         items:
           type: string
+      showInListView:
```

Comment:
nit: I would change this to `defaultShow` or something similar, the 'list view' part is coupling this pretty tightly to the UI (we may use this field on the dashboard page, for example). Also mind adding a description?

Reply:
giving it a name with 'default' in it doesn't work, because it can get updated and the UI checks it to see if it should display as a column at any given time

Reply:
Ah I see what you're saying, I was just viewing this as the initial capabilities response. 

Thinking about this more, I'm don't think we should store full `DisplayField` objects in local storage. If the capabilities endpoint updates one of the other properties (say `editable`), we don't want to be relying on a cached version of the `DisplayField`. It also limits our ability to update this model in the future, every update to this API model would require a version update and migration for the user settings. 

Instead, just store a list of 'field' strings which we selected and use that to filter the full set of `DisplayField`s. When the list is blank we default to using the display fields with this field is `true`, otherwise we ignore this field.